clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -1205,7 +1215,7 @@ public void handle(LeaveGroupResponse leaveResponse, RequestFuture<Void> future)
     }
 
     // visible for testing
-    synchronized RequestFuture<Void> sendHeartbeatRequest() {
+    public synchronized RequestFuture<Void> sendHeartbeatRequest() {
```

Comment:
I would respect the "visible for testing" comment above, and leave this package-local.

Reply:
This change is no longer valid. I was trying out some things to make the `WorkerCoordinator` test work and this came through because of that. Will remove.